cypress/e2e/smoke/create_tx.cy.js

```diff
@@ -33,9 +33,7 @@ describe('[SMOKE] Create transactions tests', () => {
 
   it('[SMOKE] Verify error message for invalid amount input', () => {
     createtx.clickOnTokenselectorAndSelectSepoliaEth()
-    createtx.verifyAmountNegativeNumber()
```

Comment:
The test to check the tooltip in case of entered nonce<current nonce was lost. We need to add it again. but with positive nonce ( test data should be also updated in the test safe)